Src/BootCamp.Chapter/Check.cs

```diff
@@ -0,0 +1,31 @@
+﻿// This file is for hunting bugs only.
+// Completing homework 3 before looking at this is HIGHLY recommended.
+// Try to look at the code in GitHub first. Try to find the mistakes that
+// were made without help or tools first.
+// After that try to find every single thing that seems off.
+// Have fun! :)
+namespace BootCamp.Chapter
+{
+    public class Checks
+    {
+        public static string PromptString(string message)
+        {
+            return Lesson3.PromptString(message);
+        }
+
+        public int PromptInt(string message)
+        {
+            return Lesson3.ReadMyInput(message);
+        }
+
+        public float PromptFloat(string message)
+        {
+            return Lesson3.AsFloat(message);
+        }
+
+        public float WhatIsTheBMI(float weight, float heigth)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename method 'WhatIsTheBMI' to match pascal case naming rules, consider using 'WhatIsTheBmi'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4886034)